### PR TITLE
feat(argo-cd): Upgrade Argo CD to v2.7.0

### PR DESCRIPTION
**Pull request by yu-croco**
release note: https://github.com/argoproj/argo-cd/releases/tag/v2.7.0
upgrade guide https://argo-cd.readthedocs.io/en/latest/operator-manual/upgrading/2.6-2.7/

---

Signed-off-by: yu-croco <yu.croco@gmail.com>

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-cd/templates/crds/crd-application.yaml [hunk 1]
```diff
@@ -310,6 +310,10 @@ spec:
                             description: CommonAnnotations is a list of additional
```
Comment (yu-croco) [inline charts/argo-cd/templates/crds/crd-application.yaml:310]: I referred https://github.com/argoproj/argo-cd/blob/v2.7.0/manifests/crds/application-crd.yaml 🙋

File: charts/argo-cd/templates/crds/crd-applicationset.yaml [hunk 2]
```diff
@@ -241,6 +241,8 @@ spec:
                                           additionalProperties:
```
Comment (yu-croco) [inline charts/argo-cd/templates/crds/crd-applicationset.yaml:241]: I referred https://github.com/argoproj/argo-cd/blob/v2.7.0/manifests/crds/applicationset-crd.yaml 🙋

**Comment by yu-croco** [comment]
I am investigating.... 🤔 
image

**Review by jmeridth — APPROVED** [review]

**Comment by pdrastil** [comment]
LGTM, however there are more changes related to Redis / Dex versions in latest relates
See history: https://github.com/argoproj/argo-cd/commits/v2.7.0/manifests

**Review by pdrastil — APPROVED** [review]

**Comment by yu-croco** [comment]
Thank you for the reference @pdrastil . I will make another PR to update 🙋

**Comment by pdrastil** [comment]
@yu-croco Another one that I've seen and relates to tini entrypoint / config that was added recently:
https://github.com/argoproj/argo-cd/commit/c2e0026de16c2622ddf4c95d0311c321ebd51f3a

**Comment by yu-croco** [comment]
oh.... I will make another PR.😇

**Pull request merged**